_components/in-page-navigation/guidance/usability.md

```diff
@@ -1,7 +1,9 @@
+- **Display the in-page navigation to the side of the main content.** Visually, place the in-page navigation component after the main content in the language’s natural reading order. (For example, for a left-to-right language like English, this component goes to the right of the main content.)
```

Comment:
Optional, but might be easier to follow without the parenthesis:

```suggestion
- **Display the in-page navigation to the side of the main content.** Visually, place the in-page navigation component after the main content in the language’s natural reading order. For example, for a left-to-right language like English, this component goes to the right of the main content.
```

Reply:
@sarah-sch, @bonnieAcameron What do you think of this suggestion?

Reply:
@bonnieAcameron, @amyleadem - I'm ok with them out

Reply:
I like getting rid of the parentheses, thumbs up

Reply:
Removed parentheses in [1ed61e9](https://github.com/uswds/uswds-site/pull/2033/commits/1ed61e9bd949a0b8c661724fba1450f7b232cfa4)